internal/pkg/manifest/lb_web_svc.go

```diff
@@ -124,19 +123,13 @@ func newDefaultLoadBalancedWebService() *LoadBalancedWebService {
 // MarshalBinary serializes the manifest object into a binary YAML document.
 // Implements the encoding.BinaryMarshaler interface.
 func (s *LoadBalancedWebService) MarshalBinary() ([]byte, error) {
-	content, err := s.parser.Parse(lbWebSvcManifestPath, *s, template.WithFuncs(map[string]interface{}{
-		"dirName": tplDirName,
-	}))
+	content, err := s.parser.Parse(lbWebSvcManifestPath, *s, template.WithFuncs(map[string]interface{}{}))
```

Comment:
We can do this I believe now!
```suggestion
	content, err := s.parser.Parse(lbWebSvcManifestPath, *s)
```

Reply:
That's true, looks like it adds nothing now. Adjusted.